tests/autoyast/verify_system_lvm.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Basic verification of system after autoyast installation
+# with lvm on btrfs.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use parent 'installbasetest';
+use testapi;
+
+sub run {
+
+    # Check if AutoYaST displayed a warning about shrinking partitions to make them
+    # fit (bsc#1078418).
+    record_info("Warning", "Warning displayed for partition shrinking")
+      unless (script_run("zgrep \"Some additional space\" /var/log/YaST2/y2log*.gz"));
+
+    # Check if all users has some value in the password field
+    # (bsc#973639, bsc#974220, bsc#971804 and bsc#965852)
+    die "Not all users' passwords are defined" unless (script_run("getent shadow | grep -E \"^[^:]+::\""));
```

Comment:
Keep in mind that `assert_script_run` supports `fail_massage`, which won't require using `die`.

Reply:
The problem with assert_script_run is that it will pas if the grep command finds the regex. Here' we don't want it to find it. If I change it to the regex found for users with defined password, I need to make sure it's there for every line, so this makes the check more complex. WDYT?

Reply:
We can always convert command to negated one, to start with. Also grep has options like `-v`, see https://stackoverflow.com/questions/3548453/negative-matching-using-grep-match-lines-that-do-not-contain-foo

Also, we can consider changing the way we assert password entries for the users, as this one is not very easy to grasp from the first sight.

Reply:
negate would help, -v not as far as I understand.